WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -220,6 +228,17 @@ public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
         MenuInflater inflater = getMenuInflater();
         inflater.inflate(R.menu.edit_post, menu);
+
+        mTags = (SuggestionAutoCompleteText) findViewById(R.id.tags);
+        mTags.setTokenizer(new SuggestionAutoCompleteText.CommaTokenizer());
+
+        remoteBlogId = WordPress.getCurrentRemoteBlogId();
+        mSuggestionServiceConnectionManager = new SuggestionServiceConnectionManager(this, remoteBlogId);
+        mTagSuggestionAdapter = SuggestionUtils.setupTagSuggestions(remoteBlogId, this, mSuggestionServiceConnectionManager);
```

Comment:
Bind service is called here but unbind is never called. Should be bound/unbound during onResume/onPause.